sci-libs/gvar/gvar-13.0.2.ebuild

```diff
@@ -0,0 +1,20 @@
+EAPI=8
+
+DISTUTILS_EXT=1
+PYTHON_COMPAT=( python3_{10..12} )
+DISTUTILS_USE_PEP517=setuptools
+inherit distutils-r1 pypi
+
+DESCRIPTION="Gaussian random variables."
+HOMEPAGE="https://github.com/gplepage/gvar"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64"
+
+RDEPEND="
+	>=dev-python/cython-0.17[${PYTHON_USEDEP}]
```

Comment:
I think (but I'm not 100% sure), that a dependency on cython is added automatically when `DISTUTILS_EXT` is set.

Reply:
`::gentoo` has > 20 packages with both https://github.com/search?q=repo%3Agentoo%2Fgentoo+DISTUTILS_EXT+cython&type=code so I'd expect it is needed.

Reply:
The eclass has this though:
```
		if [[ ${DISTUTILS_EXT} ]]; then
			packages+=(
				dev-python/cython
			)
		fi
```

Reply:
Ok, stuff like numpy then just has it to filter versions to `>cython-3.0.0`, but by now that is also satisfied by all cythons.

Reply:
Aah nevermind, this seems to be something to do with the logging, not with the dependencies: https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=677b69f23cb3c0e021a21b8a775b1da3143fd564

Reply:
Wait, that is just a debug print what you showed, right?

Reply:
Yeah it prints the version of cython, but never adds a dependency on it. So I was wrong and we do need to add the cython dependency here.

I wonder if this is intentional though.